app/scripts/components/common/map/controls/aoi/custom-aoi-control.tsx

```diff
@@ -86,6 +86,7 @@ function CustomAoI({
   // from feature to point.
   const [, forceUpdate] = useState(0);
   useEffect(() => {
+    if(!map.loaded()) return;
```

Comment:
I suspect the issue with the AOIs disappearing is related to the `flyTo` method from Mapbox and/or the way (and when) we invoke it. I've removed it in all cases where we call the `flyTo` and I'm not able reproduce AOIs disappearing at all (presets or drawn) 🤔

Reply:
I added this line because of a warning sentence from mapboxgl-draw - https://github.com/mapbox/mapbox-gl-draw/blob/main/docs/API.md#api-reference ( `Draw only works after the Mapbox GL JS map has loaded, so you must interact with Draw only after your map's load event:`) but I am actually not sure if we need this or even this hook is getting triggered after map is loaded (probably not - maybe as a byproduct.) I will check if this is needed again.

Reply:
ok we looked at this together that this line doesn't likely do anything. And I tested this with the instance that it does not break anything to remove this line. I am removing this.